JustSaying/JustSaying.csproj

```diff
@@ -6,9 +6,9 @@
     <ProjectReference Include="..\JustSaying.Models\JustSaying.Models.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.25.3" />
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.1.11" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.3.3.19" />
+    <PackageReference Include="AWSSDK.Core" Version="3.3.102.5" />
```

Comment:
Are these the _minimum_ versions that support SSE for SNS? We would want the version constraint to be as low as possible, rather than whatever the latest possible versions right now are.

Reply:
You're right, actually don't need to update since I changed it to use the SetTopicAttributesRequest and am not passing the attribute when creating the topic anymore, so I'm reverting to the previous versions